### DISCUSSION TITLE
Presence Detection Help

### DISCUSSION THREAD

**Discussion by propi62** [technyon/nuki_hub#387] [General]
Hello, i would like to use the ble presence detection to wake the nukilock. (temporary changing advertising mode for 10 minutes)
in hope the lock unlocks faster.

how is it configured?
I could only find this info:

Misc
presence/devices: List of detected bluetooth devices as CSV. Can be used for presence detection.

**Comment by iranl** [comment]
See https://github.com/technyon/nuki_hub/issues?q=is%3Aissue+is%3Aclosed+presence+detection and https://github.com/technyon/nuki_hub/issues/266 and https://github.com/technyon/nuki_hub/issues/216
